packages/dev/core/src/scene.ts

```diff
@@ -304,7 +304,13 @@ export class Scene implements IAnimatable, IClipPlanesHolder, IAssetContainer {
         this.onScenePerformancePriorityChangedObservable.notifyObservers(value);
     }
 
+    /** @internal */
+    public get _pickingAvailable() {

```

Comment:
Why not using a static and simply change the value instead of overriding a prop ?

Reply:
This is a great point. I reintroduced what was there but static is better!